zcash_client_backend/src/proto.rs

```diff
@@ -222,7 +222,7 @@ pub enum ProposalDecodingError<DbError> {
     /// A transaction identifier string did not decode to a valid transaction ID.
     TxIdInvalid(TryFromSliceError),
     /// An invalid value pool identifier was encountered.
-    ValuePoolInvalid(i32),
+    ValuePoolNotSupported(i32),
```

Comment:
This change doesn't need to be documented in the changelog because the enum is new in this release. However, in the PR the enum was introduced in (specifically in aeb405ef5d6fb9f18cdfaae3a9406d8798f0226a), it got renamed without the changelog entry being renamed. Fix the changelog addition entry to say `ProposalDecodingError` instead of `ProposalError`.